### PR TITLE
bump dependencies and prepare for releases

### PR DESCRIPTION
**Pull request by ordian**
We published a bunch of crates: https://github.com/paritytech/parity-common/releases

This is required in order to upgrade substrate.

### CODE REVIEW

**Comment by cheme** [comment]
I realized that I did not merge #111 , we don't have to merge it for this PR, but we also can (I think the base code did not change since the review).

**Comment by ordian** [comment]
@cheme good idea, let's merge #111 as well (after merge conflict is resolved)

**Comment by cheme** [comment]
yes, I am updating it (needs to update env logger version too)

File: trie-db/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [0.22.2] - 2021-01-05
+- Update `hashbrown` to 0.9. [#118](https://github.com/paritytech/trie/pull/118)
```
Comment (cheme) [inline trie-db/CHANGELOG.md:10]:
```suggestion
- Update `hashbrown` to 0.9. [#118](https://github.com/paritytech/trie/pull/118)
- Externalize test to internal test crate. [#111](https://github.com/paritytech/trie/pull/111)
```

**Review by cheme — APPROVED** [review]
Looks good to me.

File: trie-db/CHANGELOG.md (hunk 1 above)
Comment (ordian) [reply trie-db/CHANGELOG.md:10]: This change is not observable for an external user.

**Pull request merged**